### PR TITLE
choice help link from dato

### PR DESCRIPTION
**Pull request by bob-voorhoede**
- option in Dato to add a help link to a choice

### CODE REVIEW

File: src/components/modular-content/decision-tree/decision-tree.njk [hunk 1]
```diff
@@ -65,6 +65,18 @@
                   {{ indexToLetter(index, true) }}:
                 </span>
                 {{ choice.content }}
+                <template v-if="choice.helpLink" >
+                  <br>
```
Comment (Hornebom) [inline src/components/modular-content/decision-tree/decision-tree.njk]: isn't it `<br />`?

File: src/components/modular-content/decision-tree/decision-tree.njk [hunk 2]
```diff
@@ -65,6 +65,18 @@
...
+                  <a
+                    @click.stop
+                    :href="choice.helpLink.externalUrl ? choice.helpLink.externalUrl : `/${ locale }/${ choice.helpLink.internalPage.slug }`"
+                    target="_blank"
+                    rel="noopener"
+                    class="tree__content-link"
```
Comment (Hornebom) [inline src/components/modular-content/decision-tree/decision-tree.njk:75]: is that class used?

**Review by Hornebom — APPROVED** [review]

File: src/components/modular-content/decision-tree/decision-tree.njk (hunk 1 above)
Comment (bob-voorhoede) [reply src/components/modular-content/decision-tree/decision-tree.njk]: that's oooooold

File: src/components/modular-content/decision-tree/decision-tree.njk (hunk 2 above)
Comment (bob-voorhoede) [reply src/components/modular-content/decision-tree/decision-tree.njk:75]: Yes, it just does the same as the main link (put arrow in front of it, etc.)
I thought you might want to look at the styling, they need't be so similar.

**Pull request merged**